HTTP_HEADER_FORMAT.md

```diff
@@ -10,49 +10,48 @@ balancers, etc.)
 
 ## Header name
 
-`Trace-Context`
+`Trace-Context` is the name of the [Http header field](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields) being defined here. 
 
 ## Field value
 
-`base16(<version>)-<version_format>`
+The format of the value a list of data values, each value is identified by a 
+unique prefix character the defines the meaning of the value as well as its
+format.  
 
-The value will be US-ASCII encoded (which is UTF-8 compliant). Character `-` is
-used as a delimiter between fields.
+values of a particular kind need not necessarily be present, but they must be 
+in the order defined below (e.g. if the Trace-id is present it must be first)
 
-### Version
+The prefix values currently defined are
 
-Is a 1-byte representing a 8-bit unsigned integer. Version 255 reserved.
+### Prefix '*' The Trace-id 
 
-### Version = 0
+The '*' character indicates that the next 32 characters are hexadecimal digits (lower case)
+that represent a 16 byte identifier.  This identifier is meant to uniquely identify 
+new 'trace'.   The intent is that all requests that are caused directly or 
+indirection from this initial request will share this Trace-id.  It is expected that
+this identifier will be generated by selecting a 16byte random number.   
 
-#### Format
-
-`base16(<trace-id>)-base16(<span-id>)-base16(<trace-options>)`
-
-All fields are required. Character `-` is used as a delimiter between fields.
-
-#### Trace-id
-
-Is the ID of the whole trace forest. It is represented as a 16-bytes array,
-e.g., `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes 0 is considered invalid.
-
-Implementation may decide to completely ignore the trace-context if the trace-id
-is invalid.
+Example
+```
+	*4bf92f3577b34da6a3ce929d0e0e4736
+```
 
-#### Span-id
+### Prefix '-' The Span-id 
 
-Is the ID of the caller span (parent). It is represented as a 8-bytes array,
-e.g., `00f067aa0ba902b7`. All bytes 0 is considered invalid.
+The '-' prefix indicates that the next 16 characters are hexadecimal digits (lower case)
```

Comment:
if this happens in the middle of the string, it's not a prefix

Reply:
When I say prefix, I mean for the value (chunk).   Thus the 'Trace-Context' is logically a list of chunks, each value (chunk) has a prefix character that define the format and meaning of that value (chunk).